src/modules/app/route-configs/LocalNewsDetailsRouteConfig.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
```

Comment:
No need for this line

Reply:
There must be a line after flow annotation 
`Expected newline after flow annotation`